include/GMSH.cc

```diff
@@ -74,53 +74,58 @@ void GMSH::output_vertices() {
   }
 }
 
+//! \brief    Generate initial stress of each point and store
+//! \details  Get ver_stress, hor_stress, and shear (not used), write output
 void GMSH::output_stresses() {
 
-  const std::string outputfilename = "stresscheck.txt";
-  std::fstream stresscheck;
-  stresscheck.open(outputfilename, std::ios::out);
-
-  if (stresscheck.is_open()) {
+  //! Declare temp value of horizontal and vertical stresses;
+  double ver_stress;
+  double hor_stress;
 
-    //! Iterate through vector and print
+  //! ycoord contains a vector of the y-coordinates (vertical) of each point
+  std::vector<double> ycoord;
 
-    for (const auto& point : vertices_) {
-      stresscheck.setf(std::ios::fixed, std::ios::floatfield);
-      //! horizontal 2d stress
-      stresscheck << point->id() - 1 << '\t';
-      stresscheck << (0.5 *
-                      (0 - (10 * ((3 - point->coordinates().at(1)) * 22))))
-                  << '\t'
-                  //! vertical 2d stress
-                  << (0 - (10 * ((3 - point->coordinates().at(1)) * 22)))
-                  << '\t' << point->coordinates().at(2) << '\t'
-                  << point->coordinates().at(2) << '\t'
-                  << point->coordinates().at(2) << '\t'
-                  << point->coordinates().at(2) << '\n';
-    }
-    stresscheck.close();
+  for (auto const& point : vertices_) {
```

Comment:
regarding the issue of making them in class `point`, I think it's difficult to get the maximum value of the vertical coordinate right? stress is that maximum minus the point right now multiply by a factor.

For this reason I prefer to have a separate class for `initstress`.